### DISCUSSION TITLE
Open source license

### DISCUSSION THREAD

**Discussion by albanah** [asyncapi/studio#539] [Q&A]
Hello all,

I wanted to use your openSource asyncApi studio to build a new plugin for https://backstage.io/, so I wanted to ask if there is any open source license. Can you please give me more information, if I can do this?

**Comment by AceTheCreator** [comment]
cc @magicmatatjahu

**Comment by magicmatatjahu** [comment]
@albanah We use Apache 2.0 https://github.com/asyncapi/studio/blob/master/LICENSE
